pkg/kubelet/disk_manager.go

```diff
@@ -84,7 +91,20 @@ func (dm *realDiskSpaceManager) getFsInfo(fsType string, f func() (cadvisorapi.F
 }
 
 func (dm *realDiskSpaceManager) IsRuntimeDiskSpaceAvailable() (bool, error) {
-	return dm.isSpaceAvailable("runtime", dm.policy.DockerFreeDiskMB, dm.cadvisor.ImagesFsInfo)
+	return dm.isSpaceAvailable("runtime", dm.policy.DockerFreeDiskMB, dm.getRuntimeImageFs)
+}
+
+func (dm *realDiskSpaceManager) getRuntimeImageFs() (cadvisorapiv2.FsInfo, error) {
+	if dm.imageFsInfoFunc == nil {
```

Comment:
It would be better to define an imageFsInfoFunc for testing, rather than falling back to using cadvisor directly.